client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -3,112 +3,162 @@ import { Rating } from '@material-ui/lab';
+            </Grid>
+            <Typography align="center" variant="h4" color="textPrimary" className={classes.sitterName}>
+              {`${profile.firstName} ${profile.lastName}`}
+            </Typography>
+            <Typography align="center" variant="subtitle2" color="textSecondary">
+              {profile.shortDescription}
+            </Typography>
+            <Grid item className={classes.locationBox}>
               <Grid item>
-                <Avatar
-                  alt="Profile Image"
-                  src="images/fae49229659862629c1efb8dbb4329b6"
-                  className={classes.avatarSmall}
-                />
+                <LocationOn color="secondary" />
               </Grid>
               <Grid item>
-                <Typography color="textPrimary"> Reviewer Name</Typography>
+                <Typography color="textSecondary">{profile.address?.city}</Typography>
               </Grid>
             </Grid>
-            <Grid item className={classes.rating}>
-              <Typography color="textPrimary"> Rating: </Typography> <Rating value={3} name="read-only" />
+            <Grid item>
+              <Typography color="textPrimary">{`Availability: ${availability}`}</Typography>
             </Grid>
-            <Grid item className={classes.textFieldBox}>
-              <TextField
-                multiline
-                variant="outlined"
-                placeholder="Write a review"
-                rows={3}
-                className={classes.textArea}
-              />
+            <Grid item className={classes.aboutMeBox}>
+              <Typography variant="h5" color="textPrimary" className={classes.aboutMeTitle}>
+                About Me
+              </Typography>
+              <Typography variant="subtitle1" color="textPrimary">
+                {profile.description}
+              </Typography>
             </Grid>
-            <Grid item>
-              <Button variant="contained" color="secondary">
-                Submit
-              </Button>
+            <Grid item className={classes.reviewBox}>
+              <Grid item>
+                <Typography color="textPrimary">Rating and Review (0)</Typography>
+              </Grid>
+              <Box className={classes.box}></Box>
+              <Grid item>
+                {!isReviewOpen ? (
+                  <Button variant="contained" color="secondary" startIcon={<Create />} onClick={handelEvent}>
+                    Write a Review
+                  </Button>
+                ) : (
+                  <Button variant="contained" color="secondary" startIcon={<Close />} onClick={handelEvent}>
+                    Close
+                  </Button>
+                )}
+              </Grid>
             </Grid>
-          </Fragment>
-        )}
-      </Grid>
-      <RequestCard />
-    </Container>
+            {!isReviewOpen ? (
+              <Fragment></Fragment>
+            ) : (
```

Comment:
Instead of returning this empty content, you could just chain the statements like this:
`!!isReviewOpen && (
 <Fragment>
                <Grid item className={classes.userBox}>
                  <Grid item>
                    ...
                    ...`